src/components/WwwFrame/TheHeader.vue

```diff
@@ -736,7 +749,7 @@ $close-search-button-size: 2.5rem;
 	.icon {
 		width: $top-nav-font-size;
 		height: $header-height;
-		stroke: $kiva-darkgreen;
+		fill: $kiva-darkgreen;
```

Comment:
I think the 'x' icon that this is for uses stroke not fill

Reply:
Oh, I converted all strokes to fills

Reply:
Could you expand on what you did for that?

Reply:
Mainly to make them consistent. You don't need to know whether an icon is built with a stroke or a fill, it's always just a fill. 
The other reason is that I don't see other icon packages using stroke.

Reply:
Def a nice simplification... @ryan-ludwig I think you updated the svg code right? or maybe regenerated after changing in Sketch...

Reply:
@matt, yes outlined the strokes in illustrator and re-exported/Svgo'd

Reply:
☝️ That's what I was wondering, thank you